lib/deliver/itunes_connect/itunes_connect_app_icon.rb

```diff
@@ -23,7 +23,7 @@ def upload_app_icon!(app, path)
 
         Helper.log.info "Starting upload of new app icon".green
 
-        evaluate_script("$('.appversionicon > .ios7-style-icon').prev().click()") # delete button
+        evaluate_script("$('.appversionicon:not(.watchIcon) > .ios7-style-icon').prev().click()") # delete button
```

Comment:
Yea. With the watch there are two icon files. The current code deletes both, but only uploads the App Icon, not the watch icon. So I created two separate methods for it.